hms-app/src/pages/YourIdeas.tsx

```diff
@@ -56,32 +56,32 @@ export default function YourIdeas() {
       />
 
       <RelevantIdeasLoader
-        setHackathon={setHackathon}
+        setHackathon={setHackathonData}
         setRelevantIdeas={setRelevantIdeas}
         selectedHackathonId={selectedHackathonId}
         setLoading={setIsLoading}
       />
 
-      {!isLoading && (
+      {!isLoading && hackathonData.startDate > new Date(1) && (
         <div>
-          <h2>{hackathon.title}</h2>
+          <h2>{hackathonData.title}</h2>
           <h2>
-            Start Date: {new Date(hackathon.startDate).toDateString()} End Date:{' '}
-            {new Date(hackathon.endDate).toDateString()}
+            Start Date: {new Date(hackathonData.startDate).toDateString()} End
+            Date: {new Date(hackathonData.endDate).toDateString()}
```

Comment:
I would think this should be done the same way as on the `home` page.  In fact you should extract this to a function (either one that takes in a Hackathon or is part of the Hackathon's defintion) and spits out a nice consistent string